_includes/logo_header.html

```diff
@@ -0,0 +1,4 @@
+<header class="masthead text-center">
+  <a class="js-scroll-trigger" href="{{ site.baseurl }}/"><img class="logo-apply" src="{{ "/img/theplatefund_logo.png" | prepend: site.baseurl_root }}"></a>
+  <h2 class="mt-5">{% translate apply.title %}</h2>
```

Comment:
I think this is the header for the apply page, assuming this makes it show up everywhere?

Reply:
Seems so, I assume the layout will still work if you pull that out of the header on the apply page.

Reply:
good catch, fixed as you describe